modules/lib/lib-content/src/test/java/com/enonic/xp/lib/content/CreateMediaHandlerTest.java

```diff
@@ -29,11 +32,32 @@ public void testExample()
     @Test
     public void createMedia()
         throws Exception
+    {
+        Mockito.when( this.contentService.create( Mockito.any( CreateMediaParams.class ) ) ).thenAnswer( mock -> createContent( (CreateMediaParams) mock.getArguments()[0] ) );
+
+        runFunction( "/site/test/CreateMediaHandlerTest.js", "createMedia" );
+
+        final ArgumentCaptor<CreateMediaParams> argumentCaptor = ArgumentCaptor.forClass( CreateMediaParams.class );
+        Mockito.verify( this.contentService, Mockito.times( 1 ) ).create( argumentCaptor.capture() );
+
+        assertEquals( 0.5, argumentCaptor.getValue().getFocalX(), 0 );
+        assertEquals( 0.5, argumentCaptor.getValue().getFocalY(), 0 );
+    }
+
```

Comment:
Shouldn't we have a test that will check that when content is created without focal point then the properties will not be set? Because that's exactly what this bug is fixing.

Reply:
This is exactly the case, when not setting from content lib, default values in the CreateMediaHandler.java are set